Added the bohn command

kalenwallin opened the pull request:
The bohn command allows you to "bohn" a mentioned user.

Commit:
Added the bohn command

The bohn command allows you to "bohn" a mentioned user.

kalenwallin commented on run.py:
```diff
@@ -101,15 +101,15 @@ async def on_message(message):
             await message.channel.send(msg, file=discord.File(file, 'can_anyone_hear_me.gif'))
         elif commands[0].lower() == 'new':
-            msg = """ 
+            msg = """
```
Not sure what happened here.

NathanKolbas replied:
It's all good!

NathanKolbas commented on run.py:
```diff
@@ -142,6 +142,8 @@ async def on_message(message):
     - Gets the most recent tweet from DocBohn\n
 • `BohnBot new`
     - Tells you about what is new, features, improvements, and/or commands for the most recent update\n
+• `BohnBot bohn @username`
```
Move this to be alphabetically next (i.e. before execute). Also add this to the `README.md` to keep documentation up to date.

NathanKolbas commented on run.py:
```diff
@@ -155,6 +157,17 @@ async def on_message(message):
         elif commands[0].lower() == 'recent-tweet':
             msg = helper.most_recent_tweet()
             await message.channel.send(msg)
+        elif commands[0].lower() == 'bohn':
```
Move this to be alphabetically next in the if-chain (i.e. before execute).

NathanKolbas commented on run.py:
```diff
@@ -155,6 +157,17 @@ async def on_message(message):
             await message.channel.send(msg)
+        elif commands[0].lower() == 'bohn':
+            try:
+                tagged = message.mentions[0].name
+                if tagged == "kalen":
+                    await message.channel.send("I will not bohn my creator")
+                    return
+                response = ("You've been bohned " + str(tagged))
+                await message.channel.send(response)
+                return
```
Convert this to an if-else or remove the return on line 168.
```suggestion
                tagged = message.mentions[0].name
                if tagged == "kalen":
                    await message.channel.send("I will not bohn my creator")
                else:
                    response = (f"You've been bohned {tagged}")
                    await message.channel.send(response)
```

It is possible for anyone to change their name to "kalen" which will then print the secret message. If you'd like to prevent that you could check by ID (although your ID would then be in a public repo): `if message.mentions[0].id == 123456789:`. Here is how to get your ID: https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID- Up to you!

NathanKolbas commented on run.py:
```diff
@@ -155,6 +157,17 @@ async def on_message(message):
+                await message.channel.send(response)
+                return
+            except:
+                await message.channel.send("[Usage]: BohnBot bohn @username")
```
Is there a reason for "[]" around usage?

Commit:
Added the bohn command + Fixed Pull Request issues

-Bohn command allows user to "bohn" a mentioned user
-Completed the recommended code changes issued by Nathan in the pull request

NathanKolbas commented on run.py:
```diff
@@ -26,7 +26,17 @@ async def on_message(message):
+        if commands[0].lower() == 'bohn':
+            try:
+                tagged = message.mentions[0]
+                if tagged.id == "209731333067505664":
```
`tagged.id` returns back an int.

NathanKolbas commented on run.py:
```diff
@@ -26,7 +26,17 @@ async def on_message(message):
+                if tagged.id == "209731333067505664":
+                    await message.channel.send("I will not bohn my creator")
+                else:
+                    response = (f"You've been bohned {tagged.name}")
```
The parentheses aren't needed.

NathanKolbas commented:
After resolving those two comments then it LGTM!

Commit:
Removed Parenthesis and Quotations
